modin/core/storage_formats/base/query_compiler.py

```diff
@@ -87,6 +87,7 @@ def axis_setter(self, labels):
     return axis_setter
 
 
+# @decorate_all_functions(small_query_compiler)
```

Comment:
This line is commented out - are we planning to have this in the final version?

Reply:
Thanks for catching! Meant to take it out but forgot to.